Tests/SotoTests/Services/Lambda/LambdaTests.swift

```diff
@@ -12,15 +12,99 @@
 //
 //===----------------------------------------------------------------------===//
 
+/*
+
+ The testing code for the AWS Lambda function is created with the following commands:
+
+ echo "exports.handler = async (event) => { return \"hello world\" };" > lambda.js
+ zip lambda.zip lambda.js
+ cat lambda.zip | base64
+ rm lambda.zip
+ rm lambda.js
+
+ */
+
 import NIO
-import SotoLambda
 import XCTest
 
+@testable import SotoIAM
+@testable import SotoLambda
```

Comment:
As we aren't access any symbols internal to these frameworks we don't need to import them are @testable